src/Chip.js

```diff
@@ -1,23 +1,95 @@
-import React from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import cx from 'classnames';
 
-const Chip = ({ children, close, className, ...props }) => {
-  return (
-    <div className={cx('chip', className)} {...props}>
-      {children}
-      {close ? <i className="close material-icons">close</i> : null}
-    </div>
-  );
-};
+class Chip extends Component {
+  componentDidMount() {
+    const { options } = this.props;
+
+    if (options && M) {
```

Comment:
if `options` is not defined, this component shouldn't initialise?

Reply:
Nope, as matter of fact you may want to use a `Chip` but not the __[JavaScript plugin](https://materializecss.com/chips.html#basic)__.
Also, the __CSS__ is different too, `chip` when you do not want to use the plugin, `chips` when you want to.

Reply:
Cool, ok